conf/ds-present-kb-collections.yaml

```diff
@@ -65,6 +65,42 @@ config:
         prefix: 'ds.samlingsbilleder'
         base: 'ds.samlingsbilleder'
         views: *MODS_IMAGE_VIEWS
+    - radiotv:
+
+        # Temporary hack to make the Preservica XML block well-formed (it is missing a namespace declaration)
+        testdatahack: &TEST_DATA_HACK
+          regexp: '<xip:DeliverableUnit +status="([^"]+)" *>'
+          replacement: '<xip:DeliverableUnit status="$1" xmlns:xip="http://example.com/">'
```

Comment:
I've discovered that the metadata files are not only xip:DeliverableUnit. They can also be xip:Collection and xip:Manifestation. Can you make the regex catch these as well?